crates/uv-python/fetch-download-metadata.py

```diff
@@ -455,7 +464,9 @@ def sort_key(download: PythonDownload) -> tuple:
     results = {}
     for download in downloads:
         key = download.key()
-        logging.info("Found %s (%s)", key, download.flavor)
+        logging.info(
+            "Found %s%s", key, (" (%s)" % download.flavor) if download.flavor else ""
```

Comment:
Just a nit here that we were printing an empty `()` when there was no flavor (PyPy)